src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,869 @@
+    Contract:          L1StandardBridge - Soneium
+    Chain ID:          1868
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x00000000000000000000000064b5a5ed26dcb17370ff4d33a8d503f0fbd06cff
+    Raw New Value:     0x00000000000000000000000078972e88ab8bbb517a36caea23b931bab58ad3c6
+    Decoded Kind:      address
+    Decoded Old Value: 0x64B5a5Ed26DCb17370Ff4d33a8D503f0fbD06CfF
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/validation/standard/standard-versions-mainnet.toml#L38">0x78972E88Ab8BBB517a36cAea23b931BAB58AD3c6</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ----- DecodedStateDiff[55] -----
+    Who:               0xf6b1554F483C391414d3830927E3600c105AaE6E
+    Contract:          DelayedWETH - Ink
+    Chain ID:
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x00000000000000000000000071e966ae981d1ce531a7b6d23dc0f27b38409087
+    Raw New Value:     0x0000000000000000000000005e40b9231b86984b5150507046e354dbfbed3d9e
+    Decoded Kind:      address
+    Decoded Old Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/c1bcf3601dfdf72f0fd4f5bade180b9c0f94d93b/validation/standard/standard-versions-mainnet.toml#L54">0x071e966ae981d1ce531a7b6d23dc0f27b38409087</a>
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/c1bcf3601dfdf72f0fd4f5bade180b9c0f94d93b/validation/standard/standard-versions-mainnet.toml#L33">0x5e40B9231B86984b5150507046e354dbFbeD3d9e</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+                       Using OP Mainnet's <a href="https://github.com/ethereum-optimism/superchain-registry/blob/c1bcf3601dfdf72f0fd4f5bade180b9c0f94d93b/superchain/configs/mainnet/ink.toml#L63">DisputeGameFactory</a>, we can find this DelayedWETH address:
+                        - <i>cast call 0x10d7B35078d3baabB96Dd45a9143B94be65b12CD "gameImpls(uint32)(address)" 0 --rpc-url mainnet</i>
+                        - <i>cast call 0x7e87B471e96b96955044328242456427A0D49694 "weth()(address)" --rpc-url mainnet</i>
+                        returns <b>0x21429aF66058BC3e4aE4a8f2EC4531AaC433ecbC</b>
+  </code>
+</pre>
+
+# Supplementary Material
+
+## Figure 0.1: Storage Layout of OPContractsManager
+
+![OPContractsManager isRC flag set to false](./images/op-contracts-manager-storage-layout.png)
+
+## Figure 0.2: Storage Layout of AnchorStateRegistryProxy
+
+![AnchorStateRegistryProxy](./images/anchor-state-registry-storage-layout.png)
```

Comment:
You'll need to change the page to both of these images: https://github.com/ethereum-optimism/superchain-ops/pull/784/files#diff-1eff5a492090ee81aa05524b6a8380071884cc0c8e7b57ab0590b9bb10bb22ffR489